Btchap/Config/BuildSettings.swift

```diff
@@ -324,20 +366,31 @@ final class BuildSettings: NSObject {
     static let authScreenShowPhoneNumber = true
     static let authScreenShowForgotPassword = true
     static let authScreenShowCustomServerOptions = true
+    static let authScreenShowSocialLoginSection = true
```

Comment:
The authentication mechanism is completely different in Tchap than in Element. So, changing those values won't affect anything, but I agree to do this, because this might change in the future.

Reply:
Yes at this time we don't use those parameters.